lib/bitcrowd_ecto/changeset.ex

```diff
@@ -316,4 +317,112 @@ defmodule BitcrowdEcto.Changeset do
       changeset
     end
   end
+
+  if Code.ensure_loaded?(Money) do
```

Comment:
Will our lib be recompiled automatically, once `Money` is added because of the optional dep?

Reply:
no, that's the main caveat when doing this at compile-time. But IMO the better choice in this trade-off. Should I add a comment to the README pointing this out?

Reply:
Yes, please do so.

Reply:
Already done